geonames/README.md

```diff
@@ -7,10 +7,10 @@ Use Geonames service to resolve nearest populated location for given latitude, l
 
 ## Installation
 
-Pre-requisite: Register a username at geonames.org and set in `config/addon.config.php`
+Pre-requisite: Register a username at https://www.geonames.org/login and set it in `config/addon.config.php`
```

Comment:
I would prefer the link to the intro page of the service. The login page has no information about the service at all, and the link to the login page is spotted easily from the intro page.